honeybee_schema/radiance/material.py

```diff
@@ -0,0 +1,314 @@
+        description='Material modifier (default: Void).'
+        )
+
+    dependencies: List[
+        Union[Void, 'Plastic', 'Glass', 'BSDF', 'Glow', 'Light', 'Trans']
+    ] = Field(
+        default=[],
+        description='List of modifiers that this modifier depends on. '
+                    'This argument is only useful for defining advanced modifiers '
+                    'where the modifier is defined based on other modifiers '
+                    '(default: []).'
+        )
+
+
+class Plastic(ModifierBase):
+    """Radiance plastic material."""
+
+    type: constr(regex='^plastic$') = 'plastic'
+
+    r_reflectance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the red channel reflectance '
+                    '(default: 0).'
+    )
+
+    g_reflectance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the green channel reflectance '
+                    '(default: 0).'
+    )
+
+    b_reflectance: float = Field(
+        default=0.0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the blue channel reflectance '
+                    '(default: 0).'
+    )
+
+    specularity: float = Field(
+        default=0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the fraction of specularity. '
+                    'Specularity fractions greater than 0.1 are not realistic. '
+                    '(default: 0).'
+    )
+
+    roughness: float = Field(
+        default=0,
+        ge=0,
+        le=1,
+        description='A value between 0 and 1 for the roughness, specified as the '
+                    'rms slope of surface facets. Roughness greater than 0.2 are '
+                    'not realistic (default: 0).'
+    )
+
+
+class Trans(Plastic):
+    """Radiance Translucent material."""
+
+    type: constr(regex='^trans$') = 'trans'
+
+    transmitted_diff: float = Field(
+        default=0,
+        ge=0,
+        le=1,
+        description='The fraction of transmitted light that is transmitted diffusely in '
+                    'a scattering fashion (default: 0).'
+    )
+
+    transmitted_spec: float = Field(
+        default=0,
+        ge=0,
+        le=1,
+        description='The fraction of transmitted light that is not diffusely scattered '
+                    '(default: 0).'
+    )
+
+    @root_validator
+    def check_sum_fractions(cls, values):
+        """Ensure sum is less than 1."""
+        trans_diff = values.get('transmitted_diff')
+        trans_spec = values.get('transmitted_spec')
+        r_refl = values.get('r_reflectance')
+        g_refl = values.get('g_reflectance')
+        b_refl = values.get('b_reflectance')
+        summed = trans_diff + trans_spec + r_refl + g_refl + b_refl
+        assert summed <= 1, 'Sum of transmitted diffuse and specular light fractions ' \
+            'cannot be greater than 1.'
```

Comment:
I would at least include the identifier of the material and sum of the values in the error message.